### DISCUSSION TITLE
OSD for mute microphone and changing of profiles.

### DISCUSSION THREAD

**Discussion by t-AIR-e** [seerge/g-helper#78] [Ideas]
After uninstalling Armory Crate, the on-screen display icons for muting the microphone and the different profiles were also removed. I'm not sure if this is a me issue or not though. It would be nice to keep the OSD for muting the microphone though.
G14 2021

**Comment by seerge** [comment]
The app doesn't mute microphone, Asus services do :) That's why i won't be adding an OSD notification for action that app doesn't control.

**Reply by t-AIR-e** [reply]
Thanks for the reply!
Would you happen to know which app I should be downloading to bring back to OSD functionality?

**Reply by lucidnx** [reply]
Hello! I have made small script for this. Update to latest ASUS System Control Interface Driver V3.1.12.0 from asus website and try this:
[link](https://github.com/lucidnx/ASUSROGBloatServiceRemovalTool/)

I am not sure whether it will work on G14 2021, but it will do nothing, so don't worry to run it.
Let me know result please :)

**Reply by t-AIR-e** [reply]
> Hello! I have made small script for this. Update to latest ASUS System Control Interface Driver V3.1.12.0 from asus website and try this: [link](https://github.com/lucidnx/ASUSROGBloatServiceRemovalTool/)
> 
> I am not sure whether it will work on G14 2021, but it will do nothing, so don't worry to run it. Let me know result please :)

Thank you! This works with G-helper and I'm on G14 21'.

**Reply by Metalogic999** [reply]
@lucidnx Oh, nice - I may give this a try. I'm on a 2023 G14
